jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile

```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.1'],
+    base_versions: ['2018.1', '2019.1', '3.0'],
```

Comment:
@roydahan  do we need also add 2.3 here?

However, upgrade from 2.3 to 2019.1.4 isn't supported by ScyllaDB.